### PR TITLE
[INLONG-6402][Sort] Modify the metadata field of oracle connector

### LINKED ISSUE #6402: [Improve][Sort] Modify the metadata field of oracle connector
### Description

Modify the metadata field of oracle connector

### InLong Component

InLong Sort

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by e-mhui**
### Prepare a Pull Request

[INLONG-6402][Sort] Modify the metadata field of oracle connector

- Fixes #6402 

### Motivation

Modify the metadata field of oracle connector.

### CODE REVIEW

File: inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/Metadata.java [hunk 1]
```diff
@@ -54,14 +54,6 @@ default String getMetadataKey(MetaField metaField) {
             case OP_TS:
                 metadataKey = "op_ts";
                 break;
-            case DATA:
```
Comment (yunqingmoswu) [inline inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/Metadata.java:57]: Why remove it?

File: inlong-sort/sort-connectors/oracle-cdc/src/main/java/org/apache/inlong/sort/cdc/oracle/table/OracleReadableMetaData.java [hunk 2]
```diff
@@ -47,9 +47,11 @@
 /** Defines the supported metadata columns for {@link OracleTableSource}. */
 public enum OracleReadableMetaData {
 
-    /** Name of the table that contain the row. */
-    TABLE_NAME(
```
Comment (yunqingmoswu) [inline inlong-sort/sort-connectors/oracle-cdc/src/main/java/org/apache/inlong/sort/cdc/oracle/table/OracleReadableMetaData.java:51]: Would it be better to keep?

File: inlong-sort/sort-connectors/oracle-cdc/src/main/java/org/apache/inlong/sort/cdc/oracle/table/OracleReadableMetaData.java [hunk 3]
```diff
@@ -59,9 +61,12 @@ public Object read(SourceRecord record) {
                     return StringData.fromString(getMetaData(record, AbstractSourceInfo.TABLE_NAME_KEY));
                 }
             }),
-    /** Name of the schema that contain the row. */
-    SCHEMA_NAME(
```
Comment (yunqingmoswu) [inline inlong-sort/sort-connectors/oracle-cdc/src/main/java/org/apache/inlong/sort/cdc/oracle/table/OracleReadableMetaData.java:63]: Would it be better to keep?

File: inlong-sort/sort-connectors/oracle-cdc/src/main/java/org/apache/inlong/sort/cdc/oracle/table/OracleReadableMetaData.java [hunk 4]
```diff
@@ -72,9 +77,11 @@ public Object read(SourceRecord record) {
                 }
             }),
 
-    /** Name of the database that contain the row. */
-    DATABASE_NAME(
```
Comment (yunqingmoswu) [inline inlong-sort/sort-connectors/oracle-cdc/src/main/java/org/apache/inlong/sort/cdc/oracle/table/OracleReadableMetaData.java:76]: Would it be better to keep?

File: inlong-sort/sort-connectors/oracle-cdc/src/main/java/org/apache/inlong/sort/cdc/oracle/table/OracleReadableMetaData.java [hunk 5]
```diff
@@ -87,18 +94,18 @@ public Object read(SourceRecord record) {
 
     /**
      * It indicates the time that the change was made in the database. If the record is read from
-     * snapshot of the table instead of the change stream, the value is always 0.
+     * snapshot of the table instead of the binlog, the value is always 0.
      */
-    OP_TS(
```
Comment (yunqingmoswu) [inline inlong-sort/sort-connectors/oracle-cdc/src/main/java/org/apache/inlong/sort/cdc/oracle/table/OracleReadableMetaData.java:92]: Would it be better to keep?

File: inlong-sort/sort-connectors/oracle-cdc/src/main/java/org/apache/inlong/sort/cdc/oracle/table/OracleReadableMetaData.java [hunk 6]
```diff
@@ -47,6 +47,70 @@
...
+
+                @Override
+                public Object read(SourceRecord record) {
+                    return StringData.fromString(getMetaData(record, AbstractSourceInfo.DATABASE_NAME_KEY));
+                }
+            }),
+
+    /**
+     * It indicates the time that the change was made in the database. If the record is read from
+     * snapshot of the table instead of the binlog, the value is always 0.
```
Comment (gong) [inline inlong-sort/sort-connectors/oracle-cdc/src/main/java/org/apache/inlong/sort/cdc/oracle/table/OracleReadableMetaData.java:97]: binlog is concept of mysql

**Review by EMsnap — APPROVED** [review]

**Review by gong — APPROVED** [review]

**Review by yunqingmoswu — APPROVED** [review]

**Pull request merged**